packages/backend-test-utils/src/next/wiring/TestBackend.ts

```diff
@@ -16,30 +16,33 @@
 
 import { Backend, createSpecializedBackend } from '@backstage/backend-app-api';
 import {
-  createServiceFactory,
   BackendFeature,
   ExtensionPoint,
   coreServices,
   createBackendModule,
   createBackendPlugin,
+  createServiceFactory,
 } from '@backstage/backend-plugin-api';
-import { mockServices } from '../services';
 import { ConfigReader } from '@backstage/config';
 import express from 'express';
+import { mockServices } from '../services';
 // Direct internal import to avoid duplication
 // eslint-disable-next-line @backstage/no-relative-monorepo-imports
 import {
   InternalBackendFeature,
   InternalBackendRegistrations,
 } from '../../../../backend-plugin-api/src/wiring/types';
+// eslint-disable-next-line @backstage/no-forbidden-package-imports
+import { HostDiscovery } from '@backstage/backend-defaults/discovery';
 import {
   DefaultRootHttpRouter,
   ExtendedHttpServer,
   MiddlewareFactory,
   createHealthRouter,
   createHttpServer,
 } from '@backstage/backend-defaults/rootHttpRouter';
-import { HostDiscovery } from '@backstage/backend-defaults/discovery';
+// Direct internal import to avoid duplication
+// eslint-disable-next-line @backstage/no-forbidden-package-imports
```

Comment:
This is out of place, same above. Bit simpler to review in general if you avoid restructuring imports.